components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from "react";
+import DevProfile from "../DevProfile/dev-profile";
+import { ChevronDownIcon, ChevronUpIcon } from "@heroicons/react/24/solid";
+import Sparkline from "components/atoms/Sparkline/sparkline";
+import useContributorPullRequests from "lib/hooks/api/useContributorPullRequests";
+import getPullRequestsToDays from "lib/utils/get-prs-to-days";
+import { useSingleContributor } from "lib/hooks/useSingleContributor";
+import { getActivity } from "../RepoRow/repo-row";
+import useRepoList from "lib/hooks/useRepoList";
+import { getFormattedDate } from "lib/utils/date-utils";
+import { useFetchUser } from "lib/hooks/useFetchUser";
+import { classNames } from "components/organisms/RepositoriesTable/repositories-table";
+import Checkbox from "components/atoms/Checkbox/checkbox";
+import clsx from "clsx";
+
+interface ContributorListTableRow {
+  contributor: DbRepoPR;
+  topic: string;
+}
+
+function getLastContributionDate(contributions: DbRepoPR[]) {
+  if (contributions.length === 0) {
+    return "-";
+  }
+  const sortedContributions = contributions.sort((a, b) => {
+    return +new Date(b.merged_at) - +new Date(a.merged_at);
+  });
+
+  return getFormattedDate(sortedContributions[0]?.merged_at);
```

Comment:
This should display as "2 days ago" and not the full date.

Reply:
Ok...